components/layouts/nav-link/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { MouseEventHandler } from 'react'
+import { useRouter } from 'next/router'
+
+type NavLinkProps = {
+  children: any
+  onClick?: MouseEventHandler
+}
+
+const NavLink = ({ children, onClick }: NavLinkProps) => (
+  <div
+    onClick={onClick}
+    className={`flex items-center mx-2 text-gray-100 cursor-pointer h-full border-b-4 border-neutral-800 ${
+      onClick ? 'hover:border-b-4 hover:border-b-white' : ''
```

Comment:
this ternary is probably a good argument for using the classNames package. Not necessarily something we do in this PR, but a good idea for the future if we want more readable classNames